src/components/screen/Setting/index.tsx

```diff
@@ -0,0 +1,87 @@
+import { Alert, SectionList, SectionListData } from 'react-native';
+import {
+  Container,
+  HeaderContainer,
+  ItemContainer,
+  ItemIcon,
+  ItemLabel,
+  SectionHeader,
+} from './styles';
+import { DefaultNavigationProps, SettingsOption } from '../../../types';
+import {
+  DefaultTheme, withTheme,
+} from 'styled-components/native';
+import React, { useRef, useState } from 'react';
+import ChangePwView from './ChangePwView';
+import { FontAwesome } from '@expo/vector-icons';
+import { IC_FACEBOOK } from '../../../utils/Icons';
+import Modal from 'react-native-modalbox';
+import { getString } from '../../../../STRINGS';
+
+function SectionItem(option: SettingsOption, theme: DefaultTheme): React.ReactElement {
+  return (
+    <ItemContainer onPress={option.onPress} testID={option.testID}>
+      {option.icon ? <ItemIcon source={option.icon} /> : null}
+      <ItemLabel>{option.label}</ItemLabel>
+      <FontAwesome name="angle-right" size={32} color={theme.fontColor} />
+    </ItemContainer>
+  );
+}
+
+interface Props {
+  navigation: DefaultNavigationProps<'Setting'>;
+  theme: DefaultTheme;
+}
+
+function SettingScreen(props: Props): React.ReactElement {
+  const { theme } = props;
```

Comment:
Replace this with `const { theme } = useThemeContext();`

Reply:
I will change it. Thx.